codegen/set-version.sh

```diff
@@ -0,0 +1 @@
+echo "ThisBuild / version := \"$(../sdk/bin/version.sh)\"" > version.sbt
```

Comment:
Could also use an sbt-dynver setup that matches the version script, but also good to use the one script everywhere.

Reply:
Let me finalize [this](https://github.com/dwijnand/sbt-dynver/pull/188) PR and we are going to be consistent 🙂